utils/navbar.py

```diff
@@ -37,11 +37,16 @@ def display_navbar():
                             }}
                             """):
 
-        left, center, right =  st.columns([1,3,3])
+        left, center, right =  st.columns([4,1,6])
         with left:
-            center = grid(1, vertical_align="center")
+            center = grid(2, vertical_align="center")
             center.page_link(
-                "src/front.py", label="Pomorski Czarodziej", icon="🧙‍♂️", use_container_width=False)
+                "src/front.py", label="Pomorski Czarodziej", icon="🧙‍♂️", use_container_width=True)
+            gh_logo = f"""
+            <svg height=40>
```

Comment:
This could probably be a static resource, but it's not that easy to setup if I remember correctly. You can refer to https://docs.streamlit.io/develop/concepts/configuration/serving-static-files